docs/release-announcement.md

```diff
@@ -1,5 +1,11 @@
 # Release announcement
 
+## Azure Industrial IoT Platform Release 2.8.2
+We are pleased to announce the release of version 2.8.2 of our Industrial IoT Platform components as a second patch update of the 2.8 Long-Term Support (LTS) release. This release contains important security updates, bugfixes, backward compatibility issues with version 2.5.x and performance optimizations.
```

Comment:
Can the wording for backward compatibility issues be changed a bit to say that it contains fixes for those issues ? Now it reads as if we have issues with 2.5.